bank_transaction.c

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
+
+int main(void)
+{
+int current_balance,transfer_amount;
+int *ptr_current_balance, *ptr_transfer_amount;
+int sum, substact;
+int *new_balance,*new_state;
+int balance;
+
+printf("Please enter amount in source bank account: "); 
+scanf("%d",&current_balance);
+ 
+printf("please enter amount in destination bank account: ");
+scanf("%d",&transfer_amount); 
+
+ptr_transfer_amount= &current_balance; 
```

Comment:
transfer amount is pointing to current_balance - how doe that make sense?  Seems like that will lead to an error at some point.   Bad names/pointers lead to confusion on what quantity you are really dealing with